src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue12246.cs

```diff
@@ -12,19 +12,19 @@ public Issue12246(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "[Bug] iOS 14 App freezes when password is entered after email";
 
-	// [Test]
-	// [Category(UITestCategories.Entry)]
-	// public void UnfocusingPasswordDoesNotHang()
-	// {
-	// 	App.WaitForElement(Entry);
-	// 	App.WaitForElement(Password);
+	[Test]
+	[Category(UITestCategories.Entry)]
+	public void UnfocusingPasswordDoesNotHang()
```

Comment:
To avoid repeated strings, can include the original test constants in this class:

```
const string Entry = "Entry";
const string Password = "Password";
const string Success = "Success";
```